scalameta/scalameta/shared/src/test/scala/scala/meta/tests/quasiquotes/SuccessSuite.scala

```diff
@@ -851,7 +851,6 @@ class SuccessSuite extends FunSuite {
   }
 
   test("1 t\"$tpe forSome { ..$stats }\"") {
-    import scala.language.existentials
```

Comment:
This import is unnecessary here, but I'm wondering whether you have tests for language imports in Scalafix.